apps/api-extractor/src/cli/RunAction.ts

```diff
@@ -52,11 +55,45 @@ export class RunAction extends CommandLineAction {
         + ' normally be performed for a ship/production build. For example, the *.api.ts'
         + ' review file is automatically copied in a local build.'
     });
+
+    this._typescriptLibPackagePath = this.defineStringParameter({
+      parameterLongName: '--typescript-lib-package',
+      argumentName: 'PATH',
+      description: 'If specified, use typings specified in the project\'s compilerOptions -> lib option'
+        + ' from this TypeScript compiler package. This option is experimental.'
```

Comment:
Let's improve these docs.  Explain the usage scenario instead of the implementation, something like:

`By default API Extractor uses its own TypeScript compiler version to analyze your project. This can often cause compiler errors due to incompatibilites between different TS versions.  Use "--typescript-compiler" to specify the folder path for your compiler version.`